addons/project_mail_plugin/__manifest__.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+{
+    'name': 'Project Mail Plugin',
+    'version': '1.0',
+    'category': 'Sales/CRM',
+    'sequence': 5,
+    'summary': 'Turn emails received in your mailbox into leads and log their content as internal notes.',
+    'description': "Turn emails received in your mailbox into leads and log their content as internal notes.",
```

Comment:
Oopsie copy/paste? :smile:

Reply:
Yes, that was a copy/paste error :) it is now corrected.